packages/playback-core/src/index.ts

```diff
@@ -207,10 +207,17 @@ export const setupMux = (
       playerSoftwareName: player_software_name,
       playerSoftwareVersion: player_software_version,
       beaconCollectionDomain,
-      metadata,
       debug,
     } = props;
 
+    const metadata = Object.keys(props.metadata).reduce((mdata, prop) => {
```

Comment:
nitpick(blocking) Just to keep it narrow (not sure if there are valid cases to send empty strings?), we should probably do something more like:

```js
const metadata = { 
  ...props.metadata, 
  video_title: props.metadata.video_title || undefined,
};
```